VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts

```diff
@@ -85,27 +85,21 @@ export class JudgeContextMenuComponent {
         this.isDroppedDown = !this.isDroppedDown;
     }
 
-    getAdditionalText(): string {
-        return this.participant.hearingRole !== HearingRole.JUDGE ? this.getHearingRole() + this.getCaseRole() : '';
+    get isJudge(): boolean {
+        return this.participant.hearingRole === HearingRole.JUDGE;
```

Comment:
DualHost role will have to be added same here mentioned by @krishnapottekkat-hmcts

Reply:
I'm not sure how this will interact with staff member. As there was no case for this in the existing code I didn't add it here.

Reply:
As discussed this will be covered by VIH-8131